src/ssl.c

```diff
@@ -19659,6 +19691,36 @@ void wolfSSL_X509_STORE_CTX_set_time(WOLFSSL_X509_STORE_CTX* ctx,
     ctx->param->flags |= WOLFSSL_USE_CHECK_TIME;
 }
 
+/* Set the host flag in the X509_VERIFY_PARAM structure */
+void wolfSSL_X509_VERIFY_PARAM_set_hostflags(WOLFSSL_X509_VERIFY_PARAM* param,
+                                             unsigned int flags)
+{
+    if (param != NULL) {
+        param->hostFlags = flags;
+    }
+}
+
+/* Sets the expected IP address to ipasc.
+ *
+ * param is a pointer to the X509_VERIFY_PARAM structure
+ * ipasc is a NULL-terminated string with N.N.N.N for IPv4 and
+ *       HH:HH ... HH:HH for IPv6.
+ *
+ * return 1 for success and 0 for failure*/
+int wolfSSL_X509_VERIFY_PARAM_set1_ip_asc(WOLFSSL_X509_VERIFY_PARAM *param,
+        const char *ipasc)
+{
+    int ret = 0;
+
+    if (param != NULL) {
+        XSTRNCPY(param->ipasc, ipasc, WOLFSSL_MAX_IPSTR-1);
+        param->ipasc[WOLFSSL_MAX_IPSTR-1] = '\0';
+        ret = 1;
```

Comment:
WOLFSSL_SUCCESS should be used instead of 1.

Reply:
Check